src/elementary/ReactionType/src/register.hpp

```diff
@@ -48,6 +48,7 @@ ReactionType::name_dictionary{
       { ReactionType::Entry{  42, "3np",            { "z,3np", "n,3np" }, { n, n, n, p } } },
       { ReactionType::Entry{  44, "n2p",            { "z,n2p", "n,n2p" }, { n, p, p } } },
       { ReactionType::Entry{  45, "npa",            { "z,npa", "n,npa" }, { n, p, a } } },
+      { ReactionType::Entry{  50, "n(0)",           { "z,n0" }, { n } } },
```

Comment:
The ENDF manual describes this as `y,n0` where `y` is for "incident charged particles and photons", i.e., not neutrons. It specifically delineates between `z` and `y`. Do we want to follow the same?